src/pages/HomePage.tsx

```diff
@@ -0,0 +1,5 @@
+export const HomePage = () => (
+  <>
+    <h1 className="title">Home Page</h1>
+  </>
```

Comment:
Why is there a fragment?
```suggestion
    <h1 className="title">Home Page</h1>
```